front/src/modules/trainschedule/components/ManageTrainSchedule/TrainSettings.tsx

```diff
@@ -32,7 +32,9 @@ export default function TrainSettings() {
   const [name, setName] = useState<string>(nameFromStore);
   // TODO TS2 : remove departureTime when drop v1
   const [departureTime, setDepartureTime] = useState<string>(departureTimeFromStore);
-  const [startTime, setStartTime] = useState(startTimeFromStore.substring(0, 19));
+  const [startTime, setStartTime] = useState(
+    convertIsoUtcToLocalTime(startTimeFromStore).substring(0, 19)
```

Comment:
Ah, here is the logic that I assume strips the `Z` suffix? If we keep this approach, we should move the `substring()` call into `convertIsoUtcToLocalTime()` I think?